src/sql/workbench/parts/queryHistory/browser/media/queryHistoryPanel.css

```diff
@@ -0,0 +1,56 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.no-queries-message {
```

Comment:
best practice is generally to scope styles more specifically so they don't impact global scope.

Reply:
👍